aries_vcx/src/handlers/connection/invitee/states/null.rs

```diff
@@ -1,12 +1,19 @@
 use crate::handlers::connection::invitee::states::invited::InvitedState;
-use crate::messages::connection::invite::Invitation;
+use crate::messages::connection::invite::{Invitation, PairwiseInvitation, PublicInvitation};
 
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct NullState {}
 
-impl From<(NullState, Invitation)> for InvitedState {
-    fn from((_state, invitation): (NullState, Invitation)) -> InvitedState {
+impl From<(NullState, PairwiseInvitation)> for InvitedState {
```

Comment:
can't we use just the wrapping `Invitation` instead of having separate transition functions for PairwiseInvitation and PublicInvitation? The code for both seems the same

Reply:
This pattern can be found also in `aries_vcx/src/handlers/connection/inviter/states/null.rs`